src/store/index.js

```diff
@@ -18,7 +35,9 @@ export default createStore({
             displayName: payload.userName,
           })
             .then(() => {
-              router.push('/dashboard')
+              const uid = firebase.auth().currentUser.uid;
+              db.firestore().collection('users').doc(uid).set({ name: payload.userName, points: 1000 })
```

Comment:
下記のような記述が一般的なので修正しましょう。
```suggestion
              db.firestore().collection('users').doc(uid).set({ 
              name: payload.userName, 
              points: 1000 
              })
```